Makefile

```diff
@@ -26,7 +32,11 @@ GPGME_ENV = CGO_CFLAGS="$(shell gpgme-config --cflags 2>/dev/null)" CGO_LDFLAGS=
 all: tools test validate .gitvalidation
 
 build: vendor
-	@$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
+	$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
+
+cross:
+	GOOS=windows GOPATH=$(shell mktemp -d):$(GOPATH) $(MAKE) build
+	GOOS=darwin GOPATH=$(shell mktemp -d):$(GOPATH) $(MAKE) build
```

Comment:
Wouldn’t it be more accurate to run native builds? At least for macOS this seems possible, e.g. skopeo does that.  Maybe not worth the effort for Windows, sure.

Reply:
at least for a smoke test of the compile. We want this in travis.